<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdhesionFlexDlg</class>
 <widget class="QDialog" name="AdhesionFlexDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>307</width>
    <height>298</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Please define adhesion molecules</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QTableWidget" name="afTable">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="baseSize">
      <size>
       <width>256</width>
       <height>171</height>
      </size>
     </property>
     <column>
      <property name="text">
       <string>Adhesion Molecule</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Molecule</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="afMoleculeLE">
         <property name="toolTip">
          <string>Specify names of the adhesion molecules you want to use int he simulation</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="afMoleculeAddPB">
         <property name="text">
          <string>Add</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Binding Formula</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="bindingFormulaLE">
         <property name="toolTip">
          <string>This is binary function that takes atwo arguments -  Molecule1 and Molecule2. The allowed functions are those given by muParser - see http://muparser.sourceforge.net/</string>
         </property>
         <property name="text">
          <string>min(Molecule1,Molecule2)</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QPushButton" name="clearAFTablePB">
         <property name="text">
          <string>Clear Table</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okPB">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelPB">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>okPB</sender>
   <signal>clicked()</signal>
   <receiver>AdhesionFlexDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>183</x>
     <y>273</y>
    </hint>
    <hint type="destinationlabel">
     <x>112</x>
     <y>266</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelPB</sender>
   <signal>clicked()</signal>
   <receiver>AdhesionFlexDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>277</x>
     <y>271</y>
    </hint>
    <hint type="destinationlabel">
     <x>237</x>
     <y>230</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
